### ISSUE TITLE
Loading instruction timing issue

### ISSUE THREAD

**Issue by yds3** [CrystalSixone/VLN-GOAT#8]
May I ask if loading the construct_instrs function, specifically when it reaches Loading augmented data prevalent_aug_train_enc.json for pretraining..., takes a long time to complete?

**Comment by CrystalSixone** [comment]
Yes. I finish this process in about 3-5 minutes since we should load the large pre-training dataset.
